Content.Server/Radio/Components/RadioSpeakerComponent.cs

```diff
@@ -24,4 +24,8 @@ public sealed partial class RadioSpeakerComponent : Component
 
     [DataField("enabled")]
     public bool Enabled;
+
+    [DataField("IsSpeaker")]
```

Comment:
Please add a comment explaining the purpose of this new data field.

Reply:
comment added